Lesson39/savas/src/components/DialogTest.js

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
```

Comment:
You only need 'useState' here but import is importing everything from react which is not efficient. Try to never import more than you actually need.

Reply:
Actually, I get this from 'tutorialspoint' and use it, but I understood what you suggested me :)